pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -700,6 +666,14 @@ func (ds *Plugin) openDB(cfg *configuration, isReadOnly bool) (*gorm.DB, string,
 	return db, version, supportsCTE, dialect, nil
 }
 
+type gormLogger struct {
+	log logrus.FieldLogger
+}
+
+func (logger gormLogger) Print(v ...interface{}) {
+	logger.log.Debug(v...)
```

Comment:
current impl is doing a Println and uses a complex formater (https://github.com/jinzhu/gorm/blob/master/logger.go#L30) 
is not this change going to affect our outputs?

Reply:
This change didn't introduce a change in behavior. If you look at the StandardLogger implementation on our hclog adapter, the end result here is the same (and confirmed with log output).

However, the logs _are_ ugly. I'll change it to pass them through the formatter.